packages/ag-grid-theme/css/salt-ag-theme.scss

```diff
@@ -1,9 +1,46 @@
-@import "./odyssey-ag-theme-mixin";
+@import "./salt-ag-theme-mixin";
+
+div[class*="ag-theme-salt-touch"] {
+  --agGrid-row-height-default: 61px;
+  --agGrid-padding-default: 16px;
+  --agGrid-fontSize-default: 16px;
+  --agGrid-header-fontSize-default: 14px;
+  --agGrid-separatorGap-default: 20px;
+  --agGrid-cornerTag-size-default: 14px;
+}
+
+div[class*="ag-theme-salt-low"] {
+  --agGrid-row-height-default: 49px;
+  --agGrid-padding-default: 12px;
+  --agGrid-fontSize-default: 14px;
+  --agGrid-header-fontSize-default: 12px;
+  --agGrid-separatorGap-default: 16px;
+  --agGrid-cornerTag-size-default: 12px;
+}
+
+div[class*="ag-theme-salt-medium"] {
+  --agGrid-row-height-default: 37px;
+  --agGrid-padding-default: 8px;
+  --agGrid-fontSize-default: 12px;
+  --agGrid-header-fontSize-default: 11px;
+  --agGrid-separatorGap-default: 12px;
+  --agGrid-cornerTag-size-default: 10px;
+}
 
-.ag-theme-odyssey-high-light,
-.ag-theme-odyssey-medium-light,
-.ag-theme-odyssey-low-light,
-.ag-theme-odyssey-touch-light {
+div[class*="ag-theme-salt-high"] {
+  --agGrid-row-height-default: 25px;
+  --agGrid-padding-default: 8px;
+  --agGrid-fontSize-default: 11px;
+  --agGrid-header-fontSize-default: 10px;
+  --agGrid-separatorGap-default: 8px;
+  --agGrid-cornerTag-size-default: 8px;
```

Comment:
we usually avoid -default on the end of tokens but you can decide if necessary for this case

Reply:
I c/p this from grid.css. Not even using most of these, so I might just delete them.